app/views/pages/sponsor.html.haml

```diff
@@ -0,0 +1,107 @@
+= provide :title, 'Sponsor'
+
+.introduction.main-section
+  .row
+    %span.vertical-align
+      %h1
+        <b><span class="logo-h">h</span>ackEDU</b> empowers high school
+        students <b>to build things they care about</b> through
+        <b>creating</b>, <b>connecting</b>, and growing <b>student-run</b>
+        coding clubs in <b>high schools</b>.
+
+.row
+  .small-11.small-centered.columns
+    %p
+      %span.h3 hackEDU 
```

Comment:
It'd probably make more sense to put this inline with the rest of the body text. You can Google the best way to do this, but just writing `<span><h3>hackEDU</h3></span>` may not be too bad.